modules/machineset-creating.adoc

```diff
@@ -58,8 +65,10 @@ endif::ash[]
 ifdef::ash[]
 Ensure that you set the `<availabilitySet>`, `<clusterID>`, and `<role>` parameter values.
 endif::ash[]
++
```

Comment:
Looks at the published output, IMO, lines 62-67 should be steps. And line 69 could be an optional step.